deploy/kubeturbo-operator/helm-charts/kubeturbo/values.yaml

```diff
@@ -10,6 +10,7 @@ image:
   tag: 8.1.6
   pullPolicy: IfNotPresent
 #  busyboxRepository: busybox
+#  imagePullSecret: ""
```

Comment:
this may not be necessary, but if you add this, do you want to make deploy/kubeturbo/values.yaml to match this?

Reply:
@esara updated. Meanwhile, I was struggling to understand why the helm-charts are replicated at 2 places, ie. `deploy/kubeturbo` and `deploy/kubeturbo-operator/helm-charts/kubeturbo`. If that is to have the charts at a more reasonable path for users to access (`deploy/kubturbo`), a link instead might be more useful to ensure the 2 don't go out of synk for whatever reason.
Another question I had was ther version of operator-sdk you are using now and the set of steps you use to publish the same. I think it will be a good reference for us to be able to handle this if need be, and ensure we don't do stuff any different than you do it.. :-)

Reply:
you are right, if there is a way to link, we should do that (the reason it exist in both places, is because we support both helm based and operator based deployments)